### ISSUE TITLE
Bracket matching of \(x : A)

### ISSUE THREAD

**Issue by guillaumebrunerie** [mortberg/cubicaltt#84]
In the cubicaltt emacs mode, the brackets in `\(x : A)` are correctly highlighted only in the source file, not in the \*cubical\* buffer (the first parenthesis is not considered as a parenthesis), which is annoying when trying to parse a complicated answer given by cubicaltt.

**Comment by mortberg** [comment]
Indeed, I'm not an expert on emacs though... I think @david-christiansen added the cubical buffer for us? Maybe he knows how to get this to work?
